File: openapi-client/zapier/models/EmailAddressName.js
const utils = require('../utils/utils');

module.exports = {
    fields: (prefix = '', isInput = true, isArrayChild = false) => {
        const {keyPrefix, labelPrefix} = utils.buildKeyAndLabel(prefix, isInput, isArrayChild)
        return [
            {
                key: `${keyPrefix}email`,
                label: `The email address. - [${labelPrefix}email]`,
                required: true,
                type: 'string',
            },
            {
                key: `${keyPrefix}name`,
                label: `Optional display name shown to recipients (e.g. in the `From:` header). - [${labelPrefix}name]`,
                type: 'string',
            },
        ]
    },
    mapping: (bundle, prefix = '') => {
        const {keyPrefix} = utils.buildKeyAndLabel(prefix)
        return {
            'email': bundle.inputData?.[`${keyPrefix}email`],
            'name': bundle.inputData?.[`${keyPrefix}name`],
        }
    },
}

File: openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/EmailAddressName.scala

package org.openapitools.client.model


case class EmailAddressName (
    /* The email address. */
    _email: String,
    /* Optional display name shown to recipients (e.g. in the `From:` header). */
    _name: Option[String]
)
object EmailAddressName {
    def toStringBody(var_email: Object, var_name: Object) =
        s"""
        | {
        | "email":$var_email,"name":$var_name
        | }
        """.stripMargin
}
